### PR TITLE
Task/iota command

### LINKED ISSUE #2: Create IoT Agent command
Create the command that starts the IoT Agent from the command line.

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,42 @@
...
+This configuration file is a JavaScript file and contains two configuration chapters:
+* **iota**: this object stores the configuration of the Northbound of the IoT Agent, and is completely managed by the
+IoT Agent library. More information about this options can be found [here](https://github.com/telefonicaid/iotagent-node-lib#configuration).
+* **mqtt**: this object stores MQTT's specific configuration. A detailed description can be found in the next section.
+
+### MQTT configuration
+These are the currently available MQTT configuration options:
+* **host**: host of the MQTT broker.
+* **port**: port where the MQTT broker is listening.
+* **defaultKey**: default API Key to use when a device is provisioned without a Configuration.
```
Comment (AlvaroVega) [inline README.md]: Typo: Configuration -> configuration

File: config.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+config.mqtt = {
+    host: 'localhost',
+    port: 1883,
+    defaultKey: '1234'
+};
+
+config.iota = {
+    logLevel: 'DEBUG',
+    contextBroker: {
+        host: '192.168.56.101',
```
Comment (AlvaroVega) [inline config.js]: use a host like 127.0.0.1 would be more generic than 192.168.56.101

File: config.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    server: {
+        port: 4041
+    },
+    deviceRegistry: {
+        type: 'memory'
+    },
+    types: {},
+    service: 'howtoService',
+    subservice: '/howto',
+    providerUrl: 'http://192.168.56.1:4041',
```
Comment (AlvaroVega) [inline config.js]: the same above

File: bin/iotagentMqtt.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    iotAgent.start(config, function (error) {
+        if (error) {
+            logger.error(context, 'Error starting MQTT IoT Agent: [%s] Exiting process', error);
+        } else {
+            logger.info(context, 'MQTT IoT Agent started');
+        }
+    });
+}
+
+start();
```
Comment (gtorodelvalle) [inline bin/iotagentMqtt.js]: Final blank line appreciated... :)

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**